runtime/src/main/java/com/dylibso/chicory/runtime/ExternalFunction.java

```diff
@@ -0,0 +1,52 @@
+package com.dylibso.chicory.runtime;
+
+import com.dylibso.chicory.wasm.types.ValueType;
+import java.util.List;
+
+public class ExternalFunction implements ExternalValue {
+    private final WasmFunctionHandle handle;
+    private final String moduleName;
+    private final String fieldName;
+    private final List<ValueType> paramTypes;
+    private final List<ValueType> returnTypes;
+
+    public ExternalFunction(
+            WasmFunctionHandle handle,
+            String moduleName,
```

Comment:
nitpick I always forget to do: since you are at it, I would get this constructor signature aligned with the rest of `External`Things with `moduleName` and `fieldName` in first and second position.

Reply:
thanks :+1: